providers/shared/extensions/runbook_district_context/extension.ftl

```diff
@@ -0,0 +1,34 @@
+[#ftl]
+
+[@addExtension
+    id="runbook_district_context"
+    aliases=[
+        "_runbook_district_context"
+    ]
+    description=[
+        "Sets the required district context for an engine call"
+    ]
+    supportedTypes=[
+        RUNBOOK_STEP_COMPONENT_TYPE
+    ]
+/]
+
+[#macro shared_extension_runbook_district_context_runbook_setup occurrence ]
+    [#assign _context = mergeObjects(
+        _context,
+        {
+            "TaskParameters" : {
+                "GenerationFramework": getCLODeploymentFramework(),
+                "GenerationProviders": getCLODeploymentProviders()?join(","),
+                "GenerationInputSource" : getCLOInputSource(),
+                "RootDir" : getCommandLineOptions().Input.RootDir,
+                "DistrictType" : getCommandLineOptions()["Input"]["Filter"]["DistrictType"],
+                "Tenant": (getActiveLayer(TENANT_LAYER_TYPE).Name)!"",
```

Comment:
Shouldn't `Tenant`, `Account` etc drive off the config of the district?

Reply:
The active layer is the the result of evaluting the district though right?